frontend/src/components/MailItem.tsx

```diff
@@ -59,7 +66,7 @@ export default function MailItem({
       direction="row"
       justify="space-between"
       alignItems="center"
-      className={mailItem}
+      className={`${mailItem} ${!isDecided && undecidedMatch}`}
```

Comment:
We're toeing the line between doing it ourselves and wanting to do it cleanly. If we need more logic than this for class names, there's a `classnames` library that will help make this clean. Just food for thought